plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -36,45 +61,51 @@ public void addPages() {
 
   @Override
   public boolean performFinish() {
-    AnalyticsPingManager.getInstance().sendPing(
-        AnalyticsEvents.APP_ENGINE_NEW_PROJECT_WIZARD_COMPLETE,
-        AnalyticsEvents.APP_ENGINE_NEW_PROJECT_WIZARD_TYPE,
-        AnalyticsEvents.APP_ENGINE_NEW_PROJECT_WIZARD_TYPE_NATIVE);
-
-    if (config.getCloudSdkLocation() == null) {
-      File location = CloudSdkPrompter.getCloudSdkLocation(getShell());
-      if (location == null) {
-        return false;
+    if (hasAppEngineComponent) {
```

Comment:
This extra check isn't really necessary because `AppEngineStandardWizardPage` disables the finish button. WDYT?

In any case, this method returns true if `hasAppEngineComponent` is false, which doesn't sound quite right. Since this is a huge if-stmt block, I suggest the following for better readability:

```java
  if (hasAppEngineComponent) {
    return false;
  }
  ...
```